src/main/java/uk/gov/companieshouse/api/strikeoffobjections/controller/ObjectionRequestController.java

```diff
@@ -1,77 +1,75 @@
-    private static final String LOG_REQUEST_ID_KEY = LogConstants.REQUEST_ID.getValue();
-    private static final String LOG_REASON_KEY = LogConstants.REASON.getValue();
     private static final String ERIC_REQUEST_ID_HEADER = "X-Request-Id";
 
     private PluggableResponseEntityFactory responseEntityFactory;
+    private IObjectionService objectionService;
     private ApiLogger apiLogger;
 
     @Autowired
-    public ObjectionRequestController(
-            PluggableResponseEntityFactory responseEntityFactory,
-            ApiLogger apiLogger) {
-
+    public ObjectionRequestController(PluggableResponseEntityFactory responseEntityFactory, IObjectionService objectionService, ApiLogger apiLogger) {
         this.responseEntityFactory = responseEntityFactory;
+        this.objectionService = objectionService;
         this.apiLogger = apiLogger;
     }
 
-    @PostMapping("/request")
-    public ResponseEntity<ChResponseBody<ObjectionResponseBody>> createObjectionRequest(
-            @Valid @RequestBody ObjectionRequestBody requestBody,
+    @PostMapping
+    public ResponseEntity<ChResponseBody<ObjectionResponse>> createObjection(
             @PathVariable("companyNumber") String companyNumber,
-            @RequestHeader(value = ERIC_REQUEST_ID_HEADER) String requestId) {
-        
+            @RequestHeader(value = ERIC_REQUEST_ID_HEADER) String requestId
+    ) {
+        Map<String, Object> logMap = new HashMap<>();
+        logMap.put(LOG_COMPANY_NUMBER_KEY, companyNumber);
+
         apiLogger.infoContext(
-            requestId,
-            String.format("POST /request received: %s %s, %s %s, %s %s",
-                LOG_REQUEST_ID_KEY, requestId, LOG_COMPANY_NUMBER_KEY, companyNumber, LOG_REASON_KEY, requestBody.getReason()));
+                requestId,
+                "POST / request received",
+                logMap
+        );
 
         try {
-            // TODO Implement actual business logic here when stories are available
-            
-            ObjectionResponseBody objectionResponseBody = new ObjectionResponseBody();
-            objectionResponseBody.setRequestId("12345");
-            
-            return responseEntityFactory.createResponse(ServiceResult.created(objectionResponseBody));
-
-        } catch (Exception e) {            
+            String objectionId = objectionService.createObjection(requestId, companyNumber);
+            ObjectionResponse response = new ObjectionResponse(objectionId);
+            return responseEntityFactory.createResponse(ServiceResult.created(response));
+        } catch (Exception e) {
             apiLogger.errorContext(
-                requestId,
-                String.format("Error processing the Strike-Off Objection request for %s %s",
-                    LOG_COMPANY_NUMBER_KEY, companyNumber),
-                e);
+                    requestId,
+                    "Error processing the Strike-Off Objection request",
```

Comment:
"Error creating the strike-off objection request" maybe? Just a bit more specific. ('request' at end could be dropped too)